modules/ROOT/nav.adoc

```diff
@@ -80,24 +75,26 @@
  ** xref:add-another-output-transform-studio-task.adoc[Add Another Output Target]
  ** xref:move-transformations-separate-file-studio-task.adoc[Move Transformations to Separate Files]
 
-* xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
-
-* xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
- ** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
- ** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
- *** xref:pce-configuration-macos.adoc[MacOS]
- *** xref:pce-configuration-windows.adoc[Windows]
-
 * xref:login-platform-from-studio.adoc[]
  ** xref:set-credentials-in-studio-to.adoc[]
 
+* xref:set-anypoint-platform-cloud.adoc[Set Anypoint Platform Cloud]
```

Comment:
What does "set" mean in this case? Are you selecting it?

Reply:
Consider removing any hard-coded link text that's not different from the H1.

Reply:
I chose “set” as it is a “general” verb. This is an introduction to the deployment options that Studio offers. Is "Configure" a better option?